assets/airports/kbos.json

```diff
@@ -350,9 +350,9 @@
                 ["TEKKK", "EXXXX", "BLZZR"]
             ]
         },
-        "BRUWN4": {
-            "icao": "BRUWN4",
-            "name": "Broon Four",
+        "BRUWN5": {
+            "icao": "BRUWN5",
+            "name": "Broon Five",
```

Comment:
This should be `Bruin Five`, named after the Boston Bruins hockey team.

https://en.wikipedia.org/wiki/Boston_Bruins